### ISSUE TITLE
Filename parameter for offline plots?

### ISSUE THREAD

**Issue by CercaTrova** [santosjorge/cufflinks#55]
Can the filename option be passed as the local filename (HTML) to be stored on the user's computer when cufflinks is in offline mode?

**Comment by santosjorge** [comment]
Hi @CercaTrova - not sure I understand the question. Why would you require an HTML filename?

**Comment by TristanBoudreault** [comment]
I have the same workflow. Basically I generate plots but not in a notebook. I do my interactive analysis in PyCharm and when I want to see a plot, I have to generate an html file and open it with a browser. To achieve that I replaced this [line](https://github.com/santosjorge/cufflinks/blob/master/cufflinks/plotlytools.py#L1041) with the following code:
```
		if not online:
			return offline.py_offline.plot(figure, filename=filename, validate=validate)
		else:
			return py.plot(figure, sharing=sharing, filename=filename, validate=validate)
```

**Comment by TristanBoudreault** [comment]
I can make a PR if you want

**Comment by CercaTrova** [comment]
Yes that would be nice. Sorry for getting to this so late, I saw it and then forgot about it until now. My use case is the same as above, I don't use Plotly in online mode, I only use their open source library so I would like there to be an entirely offline option if possible that handles all offline capabilities. The above PR is useful to me as well as perhaps others who have the same workflow.

**Comment by LeTristanB** [comment]
Took me awhile but got around to do the PR:
https://github.com/santosjorge/cufflinks/pull/82
